_delphi_utils_python/delphi_utils/export.py

```diff
@@ -113,10 +113,12 @@ def create_export_csv(
             "geo_id",
             "val",
             "se",
+            "direction",
             "sample_size",
             "missing_val",
             "missing_se",
-            "missing_sample_size"
+            "missing_sample_size",
+            "missing_direction",
```

Comment:
question: did this break without `direction` here? I am skeptical that it is necessary. iirc, direction is a deprecated column that isn't used by any indicator.

suggestion: we never produce a `missing_direction` column anywhere, so you can remove that.

Reply:
It did break without directions. The csv for both hospital admissions and doctor visits uses direction

Reply:
alright, i see, seems fine, though i'm pretty sure `missing_direction` can be removed